test/e2e/v3/generate_test.go

```diff
@@ -28,100 +28,6 @@ import (
 	"sigs.k8s.io/kubebuilder/v2/test/e2e/utils"
 )
 
-// GenerateV2 implements a go/v2 plugin project defined by a TestContext.
-func GenerateV2(kbc *utils.TestContext) {
```

Comment:
Why are removing the e2e test to ensure that the project built with the go/v2 in the project version V3 ?

Reply:
As I listed in the first commit and in the first comment, I removed the project version `3-alpha` support for `go/v2`. The file has changed too much to allow `go/v2` support it without changing too many things.

Reply:
IMO it cannot be done. It will break the integration of Kubuilder with SDK. SDK scaffolds v2 on project version 3-alpha.